src/_internals/item/index.tsx

```diff
@@ -32,9 +32,11 @@ const StyledItem = styled(Item)`
 
   &.kirk-item--wrappable {
     flex-flow: wrap;
+    padding-top: ${space.m};
```

Comment:
I don't really understand this fix. Does it override the item padding?

```
  & {
    position: relative;
    display: flex;
    padding-top: ${space.l};
    padding-bottom: ${space.l};
```

Reply:
Yes as explain during our last call, in case of `isWrappable` the global `16px` padding need to be setted to `8px` vertically and completed by direct child to manage vertical/horizontal case display.

Reply:
I agree, I don't think this this should be changed.

Reply:
I still don't understand why :/